adserver/admin.py

```diff
@@ -774,7 +775,17 @@ class ViewAdmin(AdBaseAdmin):
     model = View
 
 
+class PublisherPayoutAdmin(admin.ModelAdmin):
+    list_display = ("amount", "publisher", "date", "modified", "created")
```

Comment:
Might show the publisher first here? 
```suggestion
    list_display = ("publisher", "amount", "date", "modified", "created")
```

Reply:
My reasoning for not doing that is that makes the publisher name the clickable link but it leads to a payout, not to the publisher. I also considered maybe displaying the PK as the clickable link. Thoughts?

Reply:
I think its probably fine as you have it, for now.

Reply:
I ended up pushing a change that linked the PK.